openhtf/io/output/print_ToConsole.py

```diff
@@ -0,0 +1,89 @@
+"""Module to display test summary on console."""
+import os
+import sys
+from openhtf import util
+from openhtf.io import test_record
+from openhtf.util import measurements as mea
+import datetime
+class PrintToConsole():
```

Comment:
Need two blank lines between top-level stuff, like between the imports and the start of this class definition. [Style guide](https://www.python.org/dev/peps/pep-0008/#blank-lines).